src/components/Editor.tsx

```diff
@@ -349,22 +348,20 @@ export const Editor = ({ document, puckConfigs }: EditorProps) => {
   return (
     <>
       {!isLoading ? (
-        <DocumentProvider value={document}>
```

Comment:
Double check that everything works when removing this. I thought Brian was saying it was necessary.

Reply:
Yeah it all works on my end. I tested by using the Dev Mode feature in YSS. Did Brian mention what broke? I don't see any problems while playing around